src/redux/books/books.js

```diff
@@ -0,0 +1,67 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [
```

Comment:
- [ ] The initial state should be an empty array. If you want to add some books you could add dispatch to use the reducer and add all books that you want to have in your list, but at this point for the project, the initialState should be empty. Use dispatch to add elements will help you to have a better understanding of how Redux works. 🐱